app/src/main/java/com/github/se/signify/ui/screens/profile/MyStatsScreen.kt

```diff
@@ -69,24 +70,23 @@ fun MyStatsScreen(
     Spacer(modifier = Modifier.height(64.dp))
 
     // Number of exercises achieved
-    StatisticsRow(
+    StatisticsColumnRow(
+        columnTestTag = "ExercisesColumn",
         rowTestTag = "ExercisesRow",
-        lineText = "Number of exercises achieved",
+        lineText = "Number of exercises achieved :",
         lineTextTag = "ExercisesText",
-        columnTextList = listOf(listOf("EASY", "${easy.value}"), listOf("HARD", "${hard.value}")),
-        columnTextSPList = listOf(listOf(12, 20), listOf(12, 20)),
-        columnTextTagList = listOf("ExercisesEasyCountBox", "ExercisesHardCountBox"))
-    Spacer(modifier = Modifier.height(12.dp))
+        statsTextList = listOf("Easy", "Medium", "Hard"),
+        statsNumberList = listOf("${easy.value}", "${medium.value}", "${hard.value}"))
+    Spacer(modifier = Modifier.height(32.dp))
 
     // Number of quests achieved
-    StatisticsRow(
+    StatisticsColumnRow(
+        columnTestTag = "QuestsColumn",
         rowTestTag = "QuestsRow",
-        lineText = "Number of quests achieved",
+        lineText = "Number of quests achieved :",
```

Comment:
Is there any situation where `:`s are not needed? Maybe you should just have them appended by default later.

Reply:
It can be concatenated in the Util function.